src/app/(pages)/fuer-eltern/schliessfaecher/page.tsx

```diff
@@ -0,0 +1,36 @@
+export default function Page() {
+    return(
+    <div>
+        <h1>Schließfächer am MCG</h1>
+        <section>
+            <div className="tutorial">
+                <h2>Wie miete ich ein Schließfach?</h2>
+                <p>
+                    Schließfächer können von Schülerinnen und Schülern gemietet werden. Die Miete beträgt 2,60 € pro Monat (31,20 €/ 12 Monate) und mit <a href="https://www.astradirect.de/schueler">Schutzbrief</a> 43,20 € pro 12 Monate.

```

Comment:
NextJS has its own optimized `<Link>` tag. I would suggest using that instead of the `<a>` tags in this file.

Reply:
Thanks! This doesn't work right now though, because the `<Link>` tag has to be imported:
```typescript
import Link from "next/link";
```

Reply:
I'

> Thanks! This doesn't work right now though, because the `<Link>` tag has to be imported:
> 
> ```ts
> import Link from "next/link";
> ```

Oopsie forgot to import. I'll change it now.